### PR TITLE
Port "Added north line to shuttle console" from Corvax

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
Ported "Added north line to shuttle console" PR by @FireNameFN: https://github.com/Corvax-Frontier/corvax-frontier-14/pull/154
I just added comments.

## Why / Balance
Greatly helps navigation

## How to test
Buy shuttle, observe thin red line pointing northward when you rotate

## Media
image
- [x] I have added screenshots/videos to this PR showcasing its changes ingame

## Breaking changes
idk

**Changelog**
:cl: erhardsteinhauer
- add: Added north line to shuttle console GUI (code by @FireNameFN)

### CODE REVIEW

File: Content.Client/Shuttles/UI/ShuttleNavControl.xaml.cs [hunk 1]
```diff
@@ -188,6 +188,9 @@ protected override void Draw(DrawingHandleScreen handle)
         // Frontier - collect blip location data outside foreach - more changes ahead
         var blipDataList = new List<BlipData>();
 
+        // Frontier Corvax: north line drawing
+        DrawNorthLine(handle, rot);
```
Comment (FireNameFN) [inline Content.Client/Shuttles/UI/ShuttleNavControl.xaml.cs:192]: You moved `DrawNorthLine()` after "our grid draw" which means north line will draw on layer above console's grid but below other grids.

Comment (ErhardSteinhauer) [reply Content.Client/Shuttles/UI/ShuttleNavControl.xaml.cs:192]: I believe it makes a bit of sense: I think that north direction shouldn't be covered by console's grid so it'll stay always visible but at the same time the north direction won't obstruct visibility of grids on the map itself.

But given how thin the lines on radar are, it doesn't matter too much in the end.

**Comment by VividPups** [comment]
If you add a line to that, Is there a way you could see if you could add an line to tell expedition ships if they are far enough from an object ot be able to FTL is that possible?

**Comment by ErhardSteinhauer** [comment]
> If you add a line to that, Is there a way you could see if you could add an line to tell expedition ships if they are far enough from an object ot be able to FTL is that possible?

Don't think it should be done this way. I believe it'll be better if FTL button on expedition menu would grey out if you can't use it.

**Review by dvir001 — APPROVED** [review]
Seems fine, going to wait for another code review.

**Comment by ErhardSteinhauer** [comment]
@FireNameFN, pinging you to make sure, if you are fine with this PR.

**Comment by FireNameFN** [comment]
> @FireNameFN, pinging you to make sure, if you are fine with this PR.

Yes, it's fine.

File: Content.Client/Shuttles/UI/BaseShuttleControl.xaml.cs [hunk 2]
```diff
@@ -115,6 +115,15 @@ protected void DrawCircles(DrawingHandleScreen handle)
         }
     }
 
+    // Frontier Corvax: north line drawing
+    protected void DrawNorthLine(DrawingHandleScreen handle, Angle angle)
+    {
+        var origin = ScalePosition(-new Vector2(Offset.X, -Offset.Y));
+        var aExtent = (angle - Math.Tau / 4).ToVec() * ScaledMinimapRadius * 1.42f;
+        var lineColor = Color.Red.WithAlpha(0.02f);
```
Comment (whatston3) [inline Content.Client/Shuttles/UI/BaseShuttleControl.xaml.cs:123]:
```suggestion
        var lineColor = Color.Red.WithAlpha(0.05f);
```

Might be my monitor setup, but the line can be difficult to make out at 2% opacity, especially in poor lighting.  5% seems more reasonable.

Below: 2% opacity (as-is).
image

Below: 5% opacity (suggested).
image

**Review by whatston3 — CHANGES_REQUESTED** [review]
Lovely change, can't wait for it to be merged in.  Might make the line more opaque, seems difficult to see on some monitors when testing locally.

**Review by Cheackraze — APPROVED** [review]

File: Content.Client/Shuttles/UI/BaseShuttleControl.xaml.cs (hunk 2 above)
Comment (Cheackraze) [reply Content.Client/Shuttles/UI/BaseShuttleControl.xaml.cs:123]: I would have to agree, my eye sight is bad enough already that the current lines are difficult for me. Everything else is good. I'd even be fine with going all the way up to 10%

**Review by whatston3 — APPROVED** [review]

**Pull request merged**